docs/source/depth_estimation.mdx

```diff
@@ -50,7 +50,7 @@ Next, check out an image with:
 Now take a look at its corresponding depth map:
 
 ```py
->>> example["depth_map"]
+>>> example["depth_map"].convert("RGB")
```

Comment:
Consider explaining why are we doing this.

Reply:
Otherwise, it will throw error. As now datatype is `float32`.

Reply:
Should I just mention that dtype is `float32` and we need `uint8` hence the `.convert('RGB')`?
Btw if we use `plt.imshow(np.array(example["depth_map"]))` it will directly show colorize depth_map but again it will require more code change.

Reply:
Yeah just mentioning it sounds good. 

Regarding the second point, could you elaborate?

Reply:
For context, both code snippets below would give the same colored depth_map.

code1: 
```py
>>> def show_depthmap(depth_map):
...    if not isinstance(depth_map, np.ndarray):
...        depth_map = np.array(depth_map)

...    d_min = np.min(depth_map)
...    d_max = np.max(depth_map)
...    depth_map = colored_depthmap(depth_map, d_min, d_max)

...    plt.imshow(depth_map.astype("uint8"))
...    plt.axis("off")
...    plt.show()
```
code2:
```py
>>> def show_depthmap(depth_map):
...    if not isinstance(depth_map, np.ndarray):
...        depth_map = np.array(depth_map)

...    plt.imshow(depth_map)
...    plt.axis("off")
...    plt.show()
```

Initially depth_map is black as we're displaying it using `example["depth_map"].convert('RGB')` (PIL). If we had used `plt.imshow(np.array(example["depth_map"])` instead, we would've seen color depth_map directly. Because `plt.imshow` applies color/colormap automatically for a 2d array. 

So, even though we initially used PIL for displaying depth_map but in the end, we're using `plt.imshow()` in `show_depth_map`. So, for a reader, it might not be clear that if we want to use `plt.imshow` in the first place then why use manual colorization?

Reply:
Thanks for explaining! 

I guess this is because of the type changes introduced in the last release. With an earlier `datasets` version, I don't think it would have been the case. No?

Reply:
Even with the previous release, we would've gotten the same result. As for the 2d array `plt.imshow()` applies colormap automatically regardless of the `dtype` and also personally I did get the same result from the previous release. 

So, should I reduce the code and get rid of `colored_depthmap`? Or mention in the comment that even though manual colorization was performed but we would've gotten the same result using `plt.imshow()` as it applies colormap automatically?

Reply:
I guess commenting might be better as that way users will know how to deal with cases when colorization cannot be perfomed / inferred automatically. WDYT?

Reply:
Yes I agree. Manual colorization is helpful when we want to save depth colors. By default, depth_map is 2D gray image.